test/core/tsi/s2a/s2a_record_protocol_test.cc

```diff
@@ -26,6 +26,7 @@
 #include "src/core/tsi/s2a/record_protocol/s2a_crypter.h"
 #include "src/core/tsi/s2a/record_protocol/s2a_crypter_util.h"
 #include "src/core/tsi/s2a/s2a_constants.h"
+#include "test/core/tsi/s2a/s2a_test_data.h"
 #include "test/core/tsi/s2a/s2a_test_util.h"
 
 /** Certain tests in this library use randomly-generated initial data. This
```

Comment:
Is this (`randomly-generated`) still the case?

Reply:
Good catch - removed these two `constexpr`'s because they are no longer used.